assets/test.yaml

```diff
@@ -1,5 +1,7 @@
 assembly_path: /lustre/scratch123/tol/resources/treeval/treeval-testdata/asccTinyTest/assembly/Pyoeliiyoelii17XNL_assembly.fa
 assembly_title: asccTinyTest
+reads_path: /lustre/scratch123/tol/resources/treeval/treeval-testdata/asccTinyTest/pacbio/
```

Comment:
Can we get rid of this one, seeing as there is pacbio_read_path in the next three lines? 
Maybe rename platform to pacbio_platform, just for unity with the other pacbio variables?

Reply:
I think we should probably remove the `pacbio_reads_path` variable and keep only the `reads_path` one. In the `https://github.com/sanger-tol/cobiontcheck/blob/develop/asg_cobiont_check.py` script I initially had a variable called `pacbio_reads_path`. I renamed it to `reads_path` when I added the options for mapping Illumina, ont and CLR reads. I kept the `pacbio_reads_path` variable in `asg_cobiont_check.py` as an alias for `reads_path`, so that the usage that James is using wouldn't have to change. But we don't need this in the nf-core version. 
`reads_type` in `https://github.com/sanger-tol/cobiontcheck/blob/develop/asg_cobiont_check.py` specifies the platform (PacBio hifi, CLR, ont or Illumina)

Reply:
Ok, 
so all pacbio_variables should be read_ variables to allow for the above. Are you ok with changing this @yumisims ?

Reply:
sounds good to me, let's sync

Reply:
done